ebpf/c/tracer-ebpf.c

```diff
@@ -455,6 +431,8 @@ static int increment_tcp_stats(struct sock *sk, struct tracer_status_t *status,
 		}
 
 		struct ipv4_tuple_t t = {};
+		t.metadata = 1;
```

Comment:
Don't hardcode this. Define something in the header, and explicitly set it for udp